Documentation/contributing/governance/commit_access.rst

```diff
@@ -10,6 +10,8 @@
          https://github.com/openvswitch/ovs/blob/master/Documentation/internals/committer-grant-revocation.rst
          https://github.com/openvswitch/ovs/blob/master/Documentation/internals/committer-responsibilities.rst
 
+..       Additional ideas have been stolen from https://github.com/envoyproxy/envoy/blob/main/GOVERNANCE.md
```

Comment:
```suggestion
..       Additional ideas have been borrowed from https://github.com/envoyproxy/envoy/blob/main/GOVERNANCE.md
```

Reply:
I don't think we're planning to give it back :stuck_out_tongue_closed_eyes: